frontend/pages/index.tsx

```diff
@@ -33,12 +50,59 @@ export default function Home(): React.ReactNode {
           Dialog!
         </Link>
       </Heading>
-      <p>{formatMessage(messages.hello)}</p>
-      <p>{formatMessage(messages.welcomeMessage)}</p>
+      <p>{f("hello")}</p>
+      <p>{f("welcomeMessage")}</p>
       <p>Current locale: {locale}</p>
       <p>Default locale: {defaultLocale}</p>
       <p>Configured locales: {JSON.stringify(locales)}</p>
 
+      <main className={styles.main}>
+        <Box bg="brand.100" w="100%" h="100%" p={10}>
+          <Text fontSize="3xl" align="center" color="white" fontWeight="bold">
+            {f("featuresTitle")}
+          </Text>
+          <br></br>
+          <br></br>
+          <SimpleGrid columns={{ sm: 2, md: 3 }} spacing={14}>
+            <FeatureCard
```

Comment:
This can be made cleaner with the use of .map!
Looking at each of the items, the only thing that differs is the icon that render. The title and description are all a variation of "feature${num}" and "feature${num}msg"

We can do something like this:

```
const features = [ BiPalette, ImWrench, FaUniversity, IoAccessibility, RiCodeSSlashLine, RiOpenSourceFill];

features.map((feature, index) => {
    return (
        <FeatureCard
              title={f(`feature${index + 1}`)}
              description={f(`featureMsg${index + 1}`)}
              icon={feature}
         />
    );
});
```

also i suggest renaming feature1msg -> featureMsg1

Reply:
Also perhaps it would be cleaner to take this entire "subsection" component of the FeaturesList and put it into a component to be imported and used in the homepage as `<FeaturesList />`